coursera/commandline.py

```diff
@@ -127,6 +127,12 @@ def parse_args(args=None):
                                 default=False,
                                 help='download quiz and exam questions. (Default: False)')
 
+    group_material.add_argument('--download-notebooks',
```

Comment:
I don't think an option is required. I added an option for quiz download only because downloading quizes means sending POST requests, which may modify user's progress state, so we let user do it deliberately. Here, if it only sends GET, I think we can grab and download as much as we can. Options here and there tend to overcomplicate the code. So, please remove (if there are only GETs).

Reply:
100% agreed.

Reply:
In this case and from my point of view it is necessary to have that option because in some cases (like the course 'python-plotting') downloading the entire notebook could take a few hours (and maybe you just want the videos and so on). Don't you think that could be better just to set it as true by default rather than remove it?

Reply:
That makes sense. I didn't know notebooks are so large, I thought they are basically Python code wrapped in a JSON. Why are they so large?

Reply:
Take a look of this example... this is just one folder from the citted course.. ^_^U
![Uploading Screenshot from 2017-11-22 09-39-39.png…]()

Reply:
Some of them only have a few files (10, 20) with python content in a json. But others include many files (for example of type csv, images for machine learning testing, etc).